include/mbedtls/ecp.h

```diff
@@ -1146,6 +1146,27 @@ int mbedtls_ecp_gen_key( mbedtls_ecp_group_id grp_id, mbedtls_ecp_keypair *key,
  */
 int mbedtls_ecp_read_key( mbedtls_ecp_group_id grp_id, mbedtls_ecp_keypair *key,
                           const unsigned char *buf, size_t buflen );
+
+/**
+ * \brief           This function exports an elliptic curve private key.
+ *
+ * \param grp_id    The ECP group identifier.
```

Comment:
I don't think this should be a parameter to this function, as it's redundant with the information contained in `key->grp`. There's an asymmetry with `mbedtls_ecp_read_key()` here: our internal representation of a private key already contains information about the curve, while a serialised representation might not.